libraries/BufferedBleSpp/nanoFramework/BufferedBleSpp.cs

```diff
@@ -0,0 +1,275 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+using System;
+using nanoFramework.Device.Bluetooth.GenericAttributeProfile;
+using System.Diagnostics;
+using System.Text;
+using nanoFramework.Device.Bluetooth.Spp;
+using System.IO;
+using System.Collections;
+using nanoFramework.Device.Bluetooth;
+
+namespace nanoFramework.Device.BufferedBleSpp
+{
+    /// <summary>
+    /// Implementation of a SPP profile which can handle large buffer sizes.
+    /// </summary>
+    public class BufferedBleSpp
+    {
+        private const int BUFFER_SIZE = 250;
+
+        private string deviceName;

```

Comment:
also better if you are using the underscore naming convention here as well for global variable. Makes it much easier to read later on